csrc/mlp_cuda.cu

```diff
@@ -75,33 +75,6 @@ static rocblas_operation hipOperationToRocOperation(hipblasOperation_t op)
     AT_ERROR("HIPBLAS_STATUS_INVALID_ENUM");
 }
```

Comment:
The whole function - hipOperationToRocOperation() Can be removed, since it is NOT used.